MPF.Library/Data/ProcessingQueue.cs

```diff
@@ -61,7 +61,7 @@ private void ProcessQueue()
                 {
                     if (this.TokenSource.IsCancellationRequested)
                         break;
-
+                    System.Threading.Thread.Sleep(10);
```

Comment:
You can up this number if you want. Half a second is probably reasonable if there was nothing going on. Honestly should have implemented this the first time around.

Reply:
I can increase if you like, but the value used seems to fix the issue fine for me. CPU usage goes down to 0 and task manager reports low power usage (down from high).

Let me know if you want it at 500ms and I can edit.